homeassistant/components/homewizard/switch.py

```diff
@@ -124,3 +131,47 @@ async def async_turn_off(self, **kwargs: Any) -> None:
     def is_on(self) -> bool:
         """Return true if switch is on."""
         return bool(self.coordinator.data["state"].switch_lock)
+
+
+class HWEnergyEnableCloudEntity(HWEnergySwitchEntity):
+    """
+    Representation of the enable cloud configuration.
+
+    Disable cloud turns off all communication to HomeWizard Cloud.
+    At this point, the device is fully local.
+    """
+
+    _attr_name = "Enable cloud"
```

Comment:
I think that this name could be better. 

Maybe “Cloud connection” ?

Reply:
Better 👍